### PR TITLE
Update index.tsx

### PR DESCRIPTION
**Pull request by xy73**

### CODE REVIEW

File: streamlit_folium/frontend/src/index.tsx [hunk 1]
```diff
@@ -91,8 +91,15 @@ function onDraw(e: any) {
 
   if (type === "circle") {
     var center: [number, number] = [layer._latlng.lng, layer._latlng.lat]
-    var radius = layer.options.radius // In km
+    var radius = layer.options.radius // In meters
```
Comment (randyzwitch) [inline streamlit_folium/frontend/src/index.tsx:94]: Just to be clear, this is just correcting the incorrect comment, right? Not changing the logic?

Comment (nsight1) [reply streamlit_folium/frontend/src/index.tsx:94]: correct, the line is the same, just noticed the comment was incorrect so changed it too.

**Comment by randyzwitch** [comment]
Let me re-do the failed tests and see if its an anomaly. Python 3.9 passes, but the other two didn't

**Comment by randyzwitch** [comment]
Yeah, it was just a GitHub CI issue. I'll merge this now, and make a new package version

**Comment by randyzwitch** [comment]
@nsight1 version 0.12.0 should now be on PyPI, and will be on conda when it works its way through their build process

**Pull request merged**